### PR TITLE
Usability page

### LINKED ISSUE #267: Create new page: "Usability"
We now have content ready for a new page on CC Open Source. The new page is called "Usability" and [the content is available here](https://docs.google.com/document/d/1MwOz1UvsO5jfR1pRQtqd1rbpzHe78tKBHf9IGbohzfA/edit?usp=sharing).

Please note that links are indicated in parentheses throughout the document. The appropriate preceding text should be hyperlinked and links should be set to open in a new window.

In terms of where the page should live, it should be:
1. Visible in the "Contribute" drop down
2. Linked from the homepage in the "Usability Testing" card. Replace the "Sign up to test" button and link with "Learn more" and a link to the new page.

### PR DESCRIPTION
**Pull request by sp35**
## Fixes

Fixes #267  by @annatuma 

## Description

This adds a new page `Usability` as discussed in the issue.
PS- The links won't open in a new tab/window as markdown doesn't support this. Probably plain HTML would have to be used to achieve this.
```
<a href="www.example.com" target="_blank">link text</a>
```

## Screenshots

Screenshot from 2020-06-16 11-56-12
Screenshot from 2020-06-16 11-56-19
Screenshot from 2020-06-16 11-56-26
Screenshot from 2020-06-16 11-56-35
Screenshot from 2020-06-16 12-06-39

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: content/contributing-code/usability/contents.lr [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+At any point in time, we’re engaging with our users through **user interviews**, where we learn more about attitudes towards our products, and dig into expansion areas we’re considering.
+
+If you’re interested in volunteering your time, we invite you to sign up to talk to us. Look at the [products and features](https://calendly.com/cc-product-design) we’re currently scheduling tests for. If you’d like time to share general thoughts, sign up for a User Interview session specifically.
+
+## What to Expect
+- After scheduling a time to talk to us, here’s what you can expect:
+- You’ll need to sign and return a [consent form](https://drive.google.com/file/d/1XWt5QDu-IR76BwsPP7OidqNncha3rHbm/view) , allowing us to record the screen during the session. We typically do screen share, and the value of the session is greatly increased when we can refer back to a recording while reviewing notes and findings.
```
Comment (annatuma) [inline content/contributing-code/usability/contents.lr:22]: There's an extra space after the "consent form" text before the comma, which should be removed.

File: content/contributing-code/usability/contents.lr [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+**Carry out the test.** During the session, have your script open on one window, and a video call with screen share open on another window. We use Google Meet (Hangouts) for our video calls.
+
+Have a notes document prepared, or better yet, enlist someone else to join the call to take notes. If (and only if) your participant has consented to screen share being recorded, be ready with software that records the screen. We often use QuickTime (if on a Mac), it is simple and free. Another option is [OBS](https://obsproject.com/), which is free, open source, and cross platform.
+
+Remember that the point of the test is to ask questions and get feedback. It is not to educate, give guidance, or explain. Whether you are running a usability test, remember that you want  to see what the user will do to complete the task when no one is around to help them. Ask them to think out loud, so that you can better understand what is clear and what is confusing, and pay close attention to how the user interacts with the prototype. If something related to your research purpose catches your attention, dive into it by asking why.
```
Comment (annatuma) [inline content/contributing-code/usability/contents.lr:54]: Typo (our fault) in the second sentence. Should say "Whenever you are..." and not "Whether you are...". There is also an extr aspace between "want" and "to" in that same sentence.

File: content/contributing-code/usability/contents.lr [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+**Thank the user.** At the end of the test, remember to thank the user. If you have the budget for it, we recommend signaling gratitude with a gift card, or other reward. Members of your community may politely decline this offer, because their reward is the ability to contribute to improving your product.
+
+**Review your findings.** Spend time with your notes, rewatching the screen recording, and deeply thinking about what you have learned.
+
+We strongly recommend reading [Rocket Surgery Made Easy by Steve Krug](http://stevekrug.com/rsme.html ). The methods we employ at CC are to a large extent based on the guidance in that book. Have you read something fantastic about usability testing? Join us in the #cc-usability channel in the [Creative Commons Slack](https://slack-signup.creativecommons.org/) and share your resources!
```
Comment (annatuma) [inline content/contributing-code/usability/contents.lr:60]: Please put the Slack channel name in back ticks, like you did in the "Participate" section when it was referenced.

**Comment by kgodey** [comment]
@sp35 thank you for your work on this! I've marked this as a "Great Contribution"

**Comment by sp35** [comment]
Thank you @kgodey

**Pull request merged**